src/components/TabBar.js

```diff
@@ -49,6 +43,7 @@ export default function BasicTabs({
   stats,
   types,
 }) {
+  // State
   const [value, setValue] = React.useState(0);
```

Comment:
Why do you have to put React.useState here? Is it because line 1 in TabBar.js?

Reply:
Yes, because someone didn't import `useState` like `import { useState } from 'react'`, but instead did `import * as React from 'react`, yes. They both work.

Reply:
While they both work, it's better to be consistent.

Reply:
Agreed. Someone likes to type `React.` more I guess.